package.json

```diff
@@ -96,9 +96,7 @@
         "development": ["last 1 Chrome versions", "last 1 Firefox versions", "last 1 Safari versions"]
     },
     "lint-staged": {
-        "*.js": ["eslint --fix", "git add"],
-        "*.ts": ["eslint --ext=.ts --fix", "git add"],
-        "*.tsx": ["eslint --ext=.tsx --fix", "git add"],
+        "*.js|ts|tsx": ["eslint --fix", "git add"],
```

Comment:
where did this syntax come from? feel like it's not doing what we think it should

im seeing something like `*.{js,ts,tsx}` on the docs

Reply:
didint know that was being used for something, thought it wa s just a label, updated in separate PR for eslint